pages/blog/_.vue

```diff
@@ -72,6 +84,14 @@
       },
       heroImage() {
         return this.hero ? this.hero.image.fields.file : null;
+      },
+
+      ...mapGetters({
+        shareUrl: 'http/canonicalUrl'
+      }),
+
+      identifier() {
+        return this.$route.params.pathMatch;
       }
```

Comment:
This identifer will need to include the hostname and something to identify the fact it's going on a blog:
```   return this.origin + '/blog/' + this.$route.params.pathMatch;```
You'll have to include the 'origin' getter from http as you do with thte canonicalUrl.